### PR TITLE
Translation of license terms (6) to Spanish

### PR DESCRIPTION
**Pull request by GemmaTuron**
## Author: 

- Gemma Turon

## Language: 

- Spanish

## Terms defined:

- License
- Creative Commons License
- CC-BY
- GNU Public License
- Hippocratic license
- MIT License

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -4704,6 +4709,11 @@
...
     def: >
       ሰዎች የለውጦቻቸውን ምንጭ እስካሰራጩ ድረስ ሶፍትዌሮችን እንደገና እንዲጠቀሙ የሚያስችል (ፈቃድ)፡፡
+  es:
+    term: "Licencia Pública GNU"
+    acronym: "GPL"
+    def: >
+    Una [licencia](#license) que permite a las usuarias re-utilizar software siempre y cuando distribuyan el origen de sus cambios 
 
 
 - slug: gradient_boosting
```
Comment (ian-flores) [inline glossary.yml:4719]: @GemmaTuron, I think the linter Action is failing because this entry does not have the correct indentation.

**Review by ian-flores — CHANGES_REQUESTED** [review]

File: glossary.yml (hunk 1 above)
Comment (GemmaTuron) [reply glossary.yml:4719]: thanks for the cath @ian-flores !

File: glossary.yml [hunk 2]
```diff
@@ -6368,6 +6386,10 @@
     term: MIT ፈቃድ
     def: >
       ሰዎች ያለምንም ገደብ ሶፍትዌሮችን እንደገና እንዲጠቀሙ የሚያስችል (ፈቃድ)(#license)።
+  es:
+    term: "Licencia MIT"
+    def: >
+      Una [licencia](#license) que permite a las usuarias re-utilizar el software sin restricciones
```
[the remark this answers is not in the document]
Comment (yabellini) [reply glossary.yml:6392]: This remove the gender mark

File: glossary.yml [hunk 3]
```diff
@@ -4704,6 +4709,11 @@
     term: የጂኤንዩ የህዝብ ፈቃድ
     def: >
       ሰዎች የለውጦቻቸውን ምንጭ እስካሰራጩ ድረስ ሶፍትዌሮችን እንደገና እንዲጠቀሙ የሚያስችል (ፈቃድ)፡፡
+  es:
+    term: "Licencia Pública GNU"
+    acronym: "GPL"
+    def: >
+    Una [licencia](#license) que permite a las usuarias re-utilizar software siempre y cuando distribuyan el origen de sus cambios
```
Comment (yabellini) [inline glossary.yml:4716]:
```suggestion
    Una [licencia](#license) que permite re-utilizar software siempre y cuando distribuyan el código funte de sus cambios. 
```

File: glossary.yml [hunk 4]
```diff
@@ -2188,7 +2190,10 @@
...
-
+  es:
+    term: Licencia Creative Commons
+    def: >
+      Conjunto de [licencias](#license) que pueden ser aplicadas a trabajos publicados. Cada licencia está formada por la concatenación de uno o más de: -BY (Atribución): las usuarias deben citar la fuente original; -SA (CompartirIgual): las usuarias deben compartir su trabajo bajo una licencia similar; -NC (NoComercial): el trabajo no se puede usar con fines comerciales sin la autorización de las personas creadoras; -ND (NoDerivaciones): no se pueden crear derivativos (por ejemplo, traducciones) sin el permiso de la creadora. Así, CC-BY-NC significa "las usuarias deben atribuir el trabajo a la creadora y no lo pueden utilizar comercialmente sin permiso". El término CC-0 (cero, no letra O), se utiliza a menudo para referirse a "sin restricciones", en el que el trabajo es de dominio público.
```
Comment (yabellini) [inline glossary.yml:2196]:
```suggestion
      Conjunto de [licencias](#license) que pueden ser aplicadas a trabajos publicados. Cada licencia está formada por la concatenación de uno o más de: -BY (Atribución): se debe citar la fuente original; -SA (CompartirIgual): se debe compartir el trabajo bajo una licencia similar; -NC (NoComercial): el trabajo no se puede usar con fines comerciales sin la autorización de las personas creadoras; -ND (NoDerivaciones): no se pueden crear trabajos derivados (por ejemplo, traducciones) sin el permiso de las personas autoras. Así, CC-BY-NC significa "se debe atribuir el trabajo a las personas creadoras y no se puede utilizar comercialmente sin permiso". El término CC-0 (cero, no letra O), se utiliza a menudo para referirse a "sin restricciones", en el que el trabajo es de dominio público.
```

Comment (yabellini) [reply glossary.yml:2196]: I will suggest using the Spanish version we can found on the official website of Creative Commons.  I don't think we are doing a better job explaining this here.

**Review by nicoguaro — APPROVED** [review]

**Review by nicoguaro — APPROVED** [review]

File: glossary.yml (hunk 1 above)
Comment (nicoguaro) [reply glossary.yml:4719]: I can't find the indentation problem. The message reads

>   Error: 4761:1 syntax error: could not find expected ':' (syntax)

But I don't see anything odd there

>- slug: gpu

**Review by elletjies — APPROVED** [review]

**Review by nicoguaro — APPROVED** [review]

**Pull request closed without merging**